pyabc/sampler/redis_eps/cli.py

```diff
@@ -65,12 +70,11 @@ def get_int(var: str):
     ana_id = analysis_id
```

Comment:
So this is  what you added to avoid several runs to somehow overlap?

Reply:
exactly. all variables are now id'd via analysis_id and time index, so the association should be unique.